domains/POAS/events/lpa-updated/schema.json

```diff
@@ -9,10 +9,69 @@
       "description": "The UID of the LPA",
       "pattern": "M(-[A-Z0-9]{4}){3}"
     },
+    "actorUID": {
+      "type": "string",
+      "description": "The UID of the actor the change relates to",
+      "pattern": "urn:opg:poas:makeregister:users:([a-z0-9]{8}-)([a-z0-9]{4}-){3}([a-z0-9]{12})"
+    },
     "changeType": {
       "type": "string",
-      "description": "The type of change to the LPA"
+      "description": "The type of change to the LPA",
+      "enum": ["CREATED", "CERTIFICATE_PROVIDER_SIGN", "ATTORNEY_SIGN", "TRUST_CORPORATION_SIGN"]
```

Comment:
if we've not started using these yet, could we have them as *_SIGNED to keep the tenses the same 👨‍🏫

Reply:
They're [passthrough from the change events](https://github.com/ministryofjustice/opg-data-lpa-store/blob/f13f291aef41dabdd2f2a6d38aa1e8be5a9a2fce/lambda/update/validate.go), so if we changed them they wouldn't make sense the other way around 😄 

We could switch to `CREATE` so they're all in present tense?